src/run_sign.py

```diff
@@ -23,7 +23,8 @@ def main():
     parser.add_argument("target", type=Path, help="Path to local manifest file or artifact directory.")
     parser.add_argument("--component", nargs="?", help="Component name")
     parser.add_argument("--type", nargs="?", help="Artifact type")
-    parser.add_argument("--sigtype", choices=ACCEPTED_SIGNATURE_FILE_TYPES, help="Type of Signature file", default=".asc")
+    parser.add_argument("--signatureType", choices=ACCEPTED_SIGNATURE_FILE_TYPES, help="Type of Signature file", default=".asc")
+    parser.add_argument("--distributionPlatform", nargs="?", help="The distribution platform", default="linux")
```

Comment:
Actually I think we should change this to match the other tools. Jenkins argument is beyond the point, but we we don't use camelCase elsewhere, e.g. https://github.com/opensearch-project/opensearch-build/blob/e45558ecca8c190f694d1fa52707a184d999c3c3/src/assemble_workflow/assemble_args.py#L19.

So I think this should be back to `--sigtype` and `--platform` or we're introducing a different pattern in Python tools.

Reply:
@abhinavGupta16 Do you have any objections we moving this back to `sigtype`? 
We would also need to change the arguments in quite some places where `signArtifacts.groovy` is called.